docs/old_saga.md

```diff
@@ -1,53 +0,0 @@
-# Previous Saga's Documentation
```

Comment:
We should also remove the entry of this file in both `README.md` and `README_ZH.md`.

Reply:
Fixed